vault/assets/dashboards/vault_overview.json

```
@@ -1,2509 +1,2512 @@
-  "layout_type": "ordered",
-  "is_read_only": true,
-  "notify_list": [],
-  "reflow_type": "fixed",
-  "widgets": [
-    {
-      "id": 0,
-      "definition": {
-        "title": "About Vault",
-        "title_align": "center",
-        "type": "group",
-        "banner_img": "/static/images/integration_dashboard/vault_hero_1.png",
-        "show_title": false,
-        "layout_type": "ordered",
-        "widgets": [
-          {
-            "id": 6856665905714958,
+    "author_name": "Datadog",
+    "description": "## Vault\n\nThis dashboard provides a high-level overview of your Vault clusters so you can monitor its performance and cluster health.\n\n- [Official Vault integration docs](https://docs.datadoghq.com/integrations/vault/)\n- [Monitoring HashiCorp Vault with Datadog](https://www.datadoghq.com/blog/monitor-hashicorp-vault-with-datadog/)\n\nClone this template dashboard to make changes and add your own graph widgets.",
+    "layout_type": "ordered",
+    "template_variables": [
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "vault_cluster",
+            "prefix": "vault_cluster"
+        },
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "api_url",
+            "prefix": "api_url"
+        },
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "auth.client_token",
+            "prefix": "@auth.client_token"
+        },
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "request.client_token_accessor",
+            "prefix": "@request.client_token_accessor"
+        },
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "request.remote_address",
+            "prefix": "@request.remote_address"
+        },
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "auth.display_name",
+            "prefix": "@auth.display_name"
+        },
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "request.mount_type",
+            "prefix": "@request.mount_type"
+        },
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "request.path",
+            "prefix": "@request.path"
+        },
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "request.operation",
+            "prefix": "@request.operation"
+        }
+    ],
+    "title": "Vault - Overview",
```

```suggestion
    "title": "Vault - Overview (OpenMetricsV2)",
```

Small nit so people are aware of the new change.